src/services/types.ts

```diff
@@ -427,6 +428,8 @@ export interface LanguageServiceHost extends GetEffectiveTypeRootsHost, MinimalR
     getParsedCommandLine?(fileName: string): ParsedCommandLine | undefined;
     /** @internal */ onReleaseParsedCommandLine?(configFileName: string, oldResolvedRef: ResolvedProjectReference | undefined, optionOptions: CompilerOptions): void;
     /** @internal */ getIncompleteCompletionsCache?(): IncompleteCompletionsCache;
+
+    jsDocParsingMode?: JSDocParsingMode;

```

Comment:
Do we really need this apart from syntax where it seems like it should be "SkipAll"

Reply:
We do, because if someone is using the ProjectService (like ts-eslint), they need to pass ProjectService the option, which then passes it to its LanguageService instances, which then are the host for their own compilations. At least, that's what I'm seeing.

Reply:
No.. this does not pass to LS .. it will use from document registry

Reply:
Ah, well, I just tried to remove it, and it just where `SyntaxTreeCache` gets its value. I suppose we could make this internal at that point.

Reply:
We should remove this instead of making internal,